### PR TITLE
podio, campainmonitor, zappier.

### PR DESCRIPTION
**Pull request by TraderStf**
See FAQ at bottom
https://www.campaignmonitor.com/pricing/

See free in https://zapier.com/terms/

On another matter:
- what about sorting the TOC?
- changing to "Team Collaboration" everything is tool here?
- make clearer note for difference between "Analytics" and "Analytics, Events and  Statistics"?

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -71,6 +71,7 @@ Table of Contents
   * https://www.blockspring.com/ - Share scripts with anyone on your team: cross language and with spreadsheet users. Free for 5 million runs a month.
   * https://helpmonks.com/ - Shared inbox for teams - Free for open source projects and non-profit organizations.
   * http://typetalk.in/ - Share and discuss ideas with your team through instant messaging on the web or on your mobile.
+  * https://podio.com/ - Organize, manage and connect everything you work with. Free for 5 users with basic plan.
```
Comment (ripienaar) [inline README.md]: there's a PR already for this one but the author is just tweaking things

File: README.md [hunk 2]
```diff
@@ -292,7 +294,8 @@ Table of Contents
    * http://cloudinary.com - Image upload, powerful manipulations, storage, and delivery for sites and apps, with libraries for Ruby, Python, Java, PHP, Objective-C and more. Perpetual free tier includes 7500 images/month, 2gb storage, 5gb bandwidth.
    * https://plot.ly - graph and share your data. Free tier includes unlimited public files and 10 private files.
    * https://transloadit.com - Handles file uploads & encoding of video, audio, images, documents. Free for open source & other do-gooders. Commercial applications get the first GB free for testdriving.
-   * https://cartodb.com - create maps and geospatial APIs from your data and public data
+   * https://cartodb.com - Create maps and geospatial APIs from your data and public data
```
Comment (ripienaar) [inline README.md]: already there

File: README.md [hunk 3]
```diff
@@ -292,7 +294,8 @@ Table of Contents
...
+   * https://zapier.com - Connect web apps to move data between them. Free 5 zap, each 15 min, 100 tasks/month.
```
Comment (ripienaar) [inline README.md]: nice, but dont think it fits in the scope being infradevs and devops

**Comment by ripienaar** [comment]
you can propose PRs to sort or clarify, soon it will be in a webapp though where it will be automatic

File: README.md (hunk 3 above)
Comment (TraderStf) [inline README.md]: ? https://zapier.com/status/
hundreds of API for integration, e.g. my soft save on box, mega, mail to xyz...

**Pull request closed without merging**